detekt/detekt-config.yml

```diff
@@ -145,7 +145,7 @@ complexity:
     thresholdInObjects: 11
     thresholdInEnums: 11
     ignoreDeprecated: false
-    ignorePrivate: false
+    ignorePrivate: true
```

Comment:
Do we want to globally disable this detekt rule for private functions? I see some benefit in limiting the number of private functions as it can uncover a class that is handling too much.

Reply:
As long as we have the 600 or so character limit on each file, the number of functions shouldn't be a problem, IMO. I'm open to reverting the change and refactoring the class too. No strong opinion on the number of functions as long as the class itself is small.

Reply:
Paired with Justin on this - there wasn't a good way to refactor the private functions. Sai brings up good points. I'm good ignoring private functions as we have other checks in place.